admin/includes/template/partials/leadInputTypes/tplProductSelect.php

```diff
@@ -1,23 +1,44 @@
 <?php
 /**
- * Created by PhpStorm.
- * User: wilt
- * Date: 30/05/16
- * Time: 21:50
+ * Admin Lead Template  - productSelect partial
+ *
+ * @package templateSystem
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id:   New in v1.6.0 $
  */
 ?>
+
 <div class="form-group">
     <?php require('includes/template/partials/' . $tplVars['leadDefinition']['inputLabelTemplate']); ?>
-    <div>
-    <?php echo zen_draw_pull_down_menu('current_category_id', zen_get_category_tree('', '', '0'), '', 'id="current_category_id"') ?>
-    <span id="products_from_categories"></span>
-    </div>
+    <?php echo zen_draw_pull_down_menu('current_category_id', zen_get_category_tree('', '', '0'), '', 'id="current_category_id" class="input-group col-sm-6"') ?>
```

Comment:
Since `0` cannot always be guaranteed to be the "top" parent category, I suspect it's better to use `zen_get_category_tree()` without any parameters in this case.